packages/nuka/src/types.ts

```diff
@@ -222,6 +225,11 @@ export interface InternalCarouselProps {
    */
   beforeSlide: (currentSlideIndex: number, endSlideIndex: number) => void;
 
+  /**
+   * Unique id attribute for the carousel which may be referenced by aria attributes.
+   */
+  carouselId: string;
```

Comment:
I went with `carouseld` to disambiguate between `id` attributes and props on other elements.